lib/code_fund_web/views/property_view.ex

```diff
@@ -10,11 +10,13 @@ defmodule CodeFundWeb.PropertyView do
     "//www.google.com/s2/favicons?domain=#{domain}"
   end
 
-  def script_embed_code(conn, %CodeFund.Schema.Property{} = property) do
-    url = "https://#{conn.host}/scripts/#{property.id}/embed.js"
+  def script_embed_url(conn, %CodeFund.Schema.Property{} = property) do
```

Comment:
@hopsoft can we pull the property id and possibly conn.host out of the struct in the function def so we don't have to match later?

``` def script_embed_url(%Plug.Conn{host: host}, %CodeFund.Schema.Property{id: id} ) ```